packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -109,19 +110,17 @@ export class IpfsTopology {
 
   private async _forceBootstrapConnection(
     ipfs: IpfsApi,
-    bootstrapList: string[]
+    bootstrapList: Multiaddr[]
   ): Promise<void> {
     // Don't want to swarm connect to ourself
     const myPeerId = (await ipfs.id()).id
     const filteredBootstrapList = bootstrapList.filter((addr) => {
-      return !addr.endsWith(myPeerId)
+      return !addr.getPeerId()?.endsWith(myPeerId)
```

Comment:
@achingbrain, I pulled down this PR and updated all the ipfs deps in the various `package.json` files to use `next`.  The tests all passed, but running `ceramic daemon` in a terminal causes this line to throw the error:
```
Connecting to peers found in 'https://raw.githubusercontent.com/ceramicnetwork/peerlist/main/testnet-clay.json'
(node:126315) UnhandledPromiseRejectionWarning: TypeError: addr.getPeerId is not a function
    at /home/spencer/js-ceramic4/packages/ipfs-topology/lib/ipfs-topology.js:105:33
    at Array.filter (<anonymous>)
    at IpfsTopology._forceBootstrapConnection (/home/spencer/js-ceramic4/packages/ipfs-topology/lib/ipfs-topology.js:103:53)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async IpfsTopology.forceConnection (/home/spencer/js-ceramic4/packages/ipfs-topology/lib/ipfs-topology.js:78:9)
    at async IpfsTopology.start (/home/spencer/js-ceramic4/packages/ipfs-topology/lib/ipfs-topology.js:81:9)
    at async Ceramic._init (/home/spencer/js-ceramic4/packages/core/lib/ceramic.js:257:13)
    at async Function.create (/home/spencer/js-ceramic4/packages/cli/lib/ceramic-daemon.js:124:9)
    at async Command.<anonymous> (/home/spencer/js-ceramic4/packages/cli/lib/bin/ceramic.js:34:5)

```

Reply:
The multiaddress in the file from there are in string and needs to be converted into a Multiaddress instance.

Reply:
Yep that was it.  Pushed a fix to this PR branch, seems to be working now.  I was able to run a node on the clay testnet, create and update a stream, and then load that stream from the clay gateway